synth.py

```diff
@@ -28,39 +29,49 @@
     bazel_target="//google/cloud/oslogin/v1:oslogin-v1-py",
     include_protos=True,
 )
-# pb2's are incorrectly generated into deeper directories, so copy separately into proto/
-s.move(
-    library,
-    excludes=[
-        "nox.py",
-        "setup.py",
-        "README.rst",
-        "docs/index.rst",
-        library / "google/cloud/oslogin_v1/proto/oslogin/**",
-        library / "google/cloud/oslogin_v1/proto/oslogin_v1/**",
-    ],
-)
-s.move(library / "google/cloud/oslogin_v1/proto/**/*.py", "google/cloud/oslogin_v1/proto")
-s.move(library / "google/cloud/oslogin/common/**/*.py", "google/cloud/oslogin_v1/proto")
+
+s.move(library / "google/cloud/oslogin_v1")
+
+s.move(library / "google/cloud/oslogin", excludes=[library / "google/cloud/oslogin/common/**/*"])
+s.move(library / "google/cloud/oslogin/common", "google/cloud/oslogin_v1/common", excludes=[library / "google/cloud/oslogin/common/services"])
+
+s.move(library / "tests/unit/gapic/oslogin_v1")
+
+s.move(library / "scripts/fixup_oslogin_v1_keywords.py")
+
+s.move(library / "docs", excludes=[library / "docs/index.rst", library / "docs/common"])
 
 s.replace(
-    "google/cloud/oslogin_v1/gapic/os_login_service_client.py",
+    "google/cloud/oslogin_v1/**/*.py",
     "google-cloud-oslogin",
     "google-cloud-os-login",
 )
 
-# Fix up imports
+for file in ["google/cloud/**/*.py", "tests/**/*.py"]:
+    s.replace(
+        file,
+        "from google.cloud.oslogin.common import common_pb2 as common",
+        "from google.cloud.oslogin_v1 import common as common"
```

Comment:
Does it work to just have
```python
from google.cloud.oslogin_v1 import common
```
?

Reply:
yes, it works, changed to `from google.cloud.oslogin_v1 import common`